syntax = "proto3";
package Enze;

message DiskBlockIndex
{
    bytes hashNext = 1;
    bytes hashPrev = 2;
    uint32 nFile    = 3;
    uint32 nBlockpos = 4;
    uint32 nHeight  = 5;
    uint32 nVersion = 6;
    uint32 nTime    =7;
    uint32 nBit     = 8;
    uint32 nNonce   = 9;
    bytes hashMerkleRoot = 10;
}
